src/events/request/RequestEventHandler.ts

```diff
@@ -24,94 +23,135 @@ export default class RequestEventHandler implements EventHandler<'message'> {
 
 	// This syntax is used to ensure that `this` refers to the `RequestEventHandler` object
 	public onEvent = async ( origin: Message ): Promise<void> => {
-		if ( origin.type !== 'DEFAULT' ) {
-			return;
-		}
-
-		if ( origin.channel instanceof TextChannel ) {
-			this.logger.info( `${ origin.author.tag } posted request ${ origin.id } in #${ origin.channel.name }` );
-		}
-
-		try {
-			await origin.reactions.removeAll();
-		} catch ( error ) {
-			this.logger.error( error );
-		}
-
-		const regex = new RegExp( `(?:${ MentionCommand.getTicketLinkRegex().source }|(${ MentionCommand.ticketPattern }))(\\?\\S+)?`, 'g' );
+		if ( !BotConfig.request.testingRequestChannels || !BotConfig.request.testingRequestChannels.includes( origin.channel.id ) ) {
+			if ( origin.type !== 'DEFAULT' ) {
+				return;
+			}
 
-		if ( BotConfig.request.noLinkEmoji && !origin.content.match( regex ) ) {
-			try {
-				await origin.react( BotConfig.request.noLinkEmoji );
-			} catch ( error ) {
-				this.logger.error( error );
+			if ( origin.channel instanceof TextChannel ) {
+				this.logger.info( `${ origin.author.tag } posted request ${ origin.id } in #${ origin.channel.name }` );
 			}
 
 			try {
-				const warning = await origin.channel.send( `${ origin.author }, your request (<${ origin.url }>) doesn't contain any valid ticket reference. If you'd like to add it you can edit your message.` );
-
-				const timeout = BotConfig.request.warningLifetime;
-				await warning.delete( { timeout } );
+				await origin.reactions.removeAll();
 			} catch ( error ) {
 				this.logger.error( error );
 			}
 
-			return;
-		}
+			const regex = new RegExp( `(?:${ MentionCommand.getTicketLinkRegex().source }|(${ MentionCommand.ticketPattern }))(\\?\\S+)?`, 'g' );
```

Comment:
This regex is now far away from where it's actually used; and it probably can get removed altogether (as it was replaced with the util function).